Makefile

```diff
@@ -35,6 +35,9 @@ PACKAGE_URL := https://www.getlantern.org
 PACKAGED_YAML := .packaged-lantern.yaml
 MANOTO_YAML := .packaged-lantern-manoto.yaml
 
+RESOURCES_DOT_GO := ./src/github.com/getlantern/flashlight/ui/resources.go
+BNS_CERT := bns.pfx
```

Comment:
My concern with having this file here would be the risk of adding it to the repo [by accident](https://github.com/getlantern/gost/issues/2), while this makes signing a bit easier it's probably not so bad to keep the actual `bns.pfx` outside this path if that lowers the risk of adding it by mistake.